.github/workflows/regression-report.yml

```diff
@@ -668,7 +668,6 @@ jobs:
             "${{ env.CACHE_KEY_MASTER }}"
             "${{ env.CACHE_KEY_CONTRIBUTION }}"
             "${{ env.CACHE_KEY_CONFIG_FILES }}"
-            "${{ env.CACHE_KEY_GENERATED_CONFIGS }}"
```

Comment:
We need to add it to this array if it set.
If not set, it means not this mode of execution, so no cleanup is required.